src/QuiqChatClient.js

```diff
@@ -47,9 +40,23 @@ class QuiqChatClient {
       HOST: this.host,
       CONTACT_POINT: this.contactPoint,
     });
+
+    window.addEventListener('online', () => {
```

Comment:
Maybe I'm not understanding this, but will this remove the requirement to call `start` from webchat?